portal-web/test/functional/com/liferay/portalweb/macros/analyticscloud/ACIndividualsDashboard.macro

```diff
@@ -0,0 +1,17 @@
+definition {
```

Comment:
hey @ctang3 , I think you forgot to update the file path reference in these macros... 

You are calling selectors from "IndividualsDetailsPage#", however, your path file here in this PR is "ACIndividualsDashboard.path"